README.md

```diff
@@ -140,6 +140,15 @@ See all possible Heroku flags:
 
     heroku help create
 
+In order to successfully push app to heroku, following environment variables must be set:
+
+* HOST
+* ASSET_HOST or APPLICATION_HOST
```

Comment:
`APPLICATION_HOST` is required, and that renders `ASSET_HOST` optional. I'd remove it from this list.